app/src/test/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreenViewModelTest.kt

```diff
@@ -357,11 +359,12 @@ class OtherUserProfileScreenViewModelTest {
                 GetUserInfoResult.Success(OTHER_USER.copy(connectionStatus = ConnectionState.ACCEPTED), TEAM)
             )
             .withGetOneToOneConversation(CreateConversationResult.Failure(Unknown(RuntimeException("some error"))))
+            .withGetConversationDetails(GetOneToOneConversationUseCase.Result.Failure)
             .arrange()
 
         // then
         coVerify {
-            arrangement.getOrCreateOneToOneConversation(USER_ID)
+            arrangement.getConversationUseCase(USER_ID)
```

Comment:
Should we somehow check that `getOrCreateOneOnOneConversation` doesn't get invoked?

Reply:
Good catch ! Better explicit than implicit, as the Python zen would say :D